db/migrate/20190605205526_add_reason_to_issues.rb

```diff
@@ -0,0 +1,31 @@
+Dir[Rails.root.join("app/serializers/**/*.rb")].each {|f| require f}
+class AddReasonToIssues < ActiveRecord::Migration[5.2]
+  def change
+    add_column :issues, :reason_id, :integer, null:true
+
+    # 1 -> first issue of every person has new_client reason
+    Person.all.each do |p| 
+      issue = p.issues.first
+      next if issue.nil?
+      issue.update_attribute(:reason, IssueReason.new_client) 
+    end
+
+    # 5 -> issue with a risk score has new_risk_information reason
+    Issue.joins(:risk_score_seeds)
```

Comment:
Issue.joins(:risk_score_seeds).where('reason_id is null').update_all(:reason_id, IssueReason.new_risk_information.id)